index.js

```diff
@@ -0,0 +1,141 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true,
+    },
+    sourceType: 'module',
+    project: './tsconfig.json',
+  },
+  extends: ['plugin:react/recommended', 'plugin:prettier/recommended'],
+  plugins: ['import', 'unicorn', 'prettier'],
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  env: {
+    browser: true,
+    es6: true,
+  },
+  globals: {
+    require: true,
+    process: true,
+  },
+  rules: {
+    'default-case': 'error',
+    'import/extensions': [
+      'error',
+      'always',
+      { js: 'never', jsx: 'never', ts: 'never', tsx: 'never', hbs: 'never', vue: 'never' },
+    ],
+    'import/no-unresolved': 'error',
```

Comment:
I like this import ordering, is auto-fixable.

```suggestion
    'import/no-unresolved': 'error',
    'import/order': ['error', { groups: ['external', 'builtin', ['sibling', 'parent']] }],    
```

Reply:
As long as it is auto-fixable, I like it too. 

Although I don't really care about the order of my imports. It has no effect on the functionality and my editor automatically adds them for me.

But I can imagine that is can help solving merge conflicts.